Fundamentos/4- JavaScript/4.3/exbonus04.js

```diff
@@ -0,0 +1,20 @@
+let n = 5;
+let meio = (n + 1) / 2;
+let ast = "*";
+let esp = "";
+let esq = meio;
+let dir = meio;
+
+for (let i = 0; i <= meio; i += 1) {
+  for (let ind = 0; ind <= n; ind++) {
```

Comment:
uma boa prática para 'nested for' é usar índice i para o for externo, e j para o interno